lib/metrics/amplitude.js

```diff
@@ -22,6 +22,10 @@ const P = require('../promise')
 
 const APP_VERSION = /^[0-9]+\.([0-9]+)\./.exec(require('../../package.json').version)[1]
 
+const DAY = 1000 * 60 * 60 * 24
+const WEEK = DAY * 7
+const MONTH_ISH = DAY * 30
```

Comment:
Lol at the name, but also, I wonder if it's worth using 28 day here to match what we do in e.g. MAU calculations such as https://sql.telemetry.mozilla.org/queries/1312/source#2284

Reply:
I chose 30 days after searching to see if there was a commonly-accepted number to use when calculating MAU. Not sure if it counts for anything but all the results I checked from Google mention 30 days:

* http://www.adweek.com/digital/application-insights-how-exactly-is-mau-calculated/
* https://www.quora.com/What-is-the-best-way-to-calculate-MAU-monthly-active-users
* https://en.wikipedia.org/wiki/Monthly_active_users
* https://amplitude.com/blog/2016/01/14/measuring-active-users/
* http://blog.popcornmetrics.com/5-user-engagement-metrics-for-growth/

Given the wider-than-just-our-team audience that uses Amplitude, I figured it was better to do the conventional/expected thing than be consistent with what we'd done before.

Happy to change it though if you think the benefits of consistency are more important?

Reply:
(obviously the best thing to be consistent with would be whatever value for MAU the rest of Mozilla uses, but I couldn't find that documented anywhere)

Reply:
> obviously the best thing to be consistent with would be whatever value for
> MAU the rest of Mozilla uses, but I couldn't find that documented anywhere

I have in my head that it's 28, but I could easily have made that up.  Sounds like an @davismtl question to me :-)